includes/class-content-template.php

```
@@ -119,16 +119,40 @@ public function render_qa_view($content)
     $is_primary_meta = get_post_meta($post_id, "_mp_is_primary_turn", true);
     $is_primary = $is_primary_meta === '' || $is_primary_meta === "1"; // Handle empty (legacy) or explicitly "1" as primary
 
-    $html = '<div id="mp-hover-card"></div>';
-    $html .= '<div class="mp-qa-container">';
-
-    // Emit the single turn array into the DOM for hover JS dependencies
-    $html .= '<script type="application/json" id="mp-qa-content">' . wp_json_encode([$single_turn]) . "</script>";
+    $html = '<div class="mp-qa-container">';
 
     $question = $this->clean_lit_comments($single_turn["question"] ?? "");
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$question'. (undefined)

(UnusedLocalVariable)